scripts/verify-macos-selfsigned.sh

```
@@ -0,0 +1,288 @@
+# framework cannot import ("MAC verification failed"). -legacy restores the
+# algorithms it understands; LibreSSL (/usr/bin/openssl) does not need it.
+if ! openssl pkcs12 -export -legacy -out "$WORK/cert.p12" -inkey "$WORK/key.pem" \
+     -in "$WORK/cert.pem" -passout "pass:$KC_PASS" >/dev/null 2>&1; then
+  openssl pkcs12 -export -out "$WORK/cert.p12" -inkey "$WORK/key.pem" \
+    -in "$WORK/cert.pem" -passout "pass:$KC_PASS" >/dev/null 2>&1
+fi
+
+security create-keychain -p "$KC_PASS" "$KEYCHAIN" >/dev/null
+security unlock-keychain -p "$KC_PASS" "$KEYCHAIN" >/dev/null
+security set-keychain-settings "$KEYCHAIN"
+security list-keychains -d user -s "$KEYCHAIN" $(security list-keychains -d user | tr -d '"')
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)